retroshare.pri

```diff
@@ -836,25 +804,46 @@ macx-* {
 	}
 	rs_macos10.14 {
 		message(***retroshare.pri: Set Target and SDK to MacOS 10.14 )
-		QMAKE_MACOSX_DEPLOYMENT_TARGET=10.14
+		QMAKE_MACOSX_DEPLOYMENT_TARGET=10.13
```

Comment:
when rs_macos10.14 is asked, you should use 10.14. Why did you change all the versions (here and below??)